### PR TITLE
lis again

### PR DESCRIPTION
**Pull request by SansPapyrus683**
_If any of the below doesn't apply to this Pull Request, mark the checkbox as
done._

- [x] I have tested my code
- [x] I have followed the code style guidelines mentioned
      [here](https://usaco.guide/general/adding-solution/#code-conventions)
- [x] I have properly formatted the files according to the steps mentioned
      [here](https://usaco.guide/general/adding-solution#steps)

### CODE REVIEW

**Review by ryanchou-dev — APPROVED** [review]
lgtm

File: content/4_Gold/LIS.mdx [hunk 1]
```diff
@@ -47,45 +47,66 @@ $A[i]$. We can then naively compute $dp$ (and thus the LIS) in
...
 <LanguageSection>
-
 <CPPSection>
 
 ```cpp
+#include <vector>
+using namespace std;
+
 int find_lis(vector<int> a) {
-	int n = a.size(), lis = 0;
```
Comment (nwatx) [inline content/4_Gold/LIS.mdx:55]: why remove n?

File: content/4_Gold/LIS.mdx [hunk 2]
```diff
@@ -47,45 +47,66 @@ $A[i]$. We can then naively compute $dp$ (and thus the LIS) in
...
 ```
 
 </CPPSection>
-
 <JavaSection>
 
 ```java
-
-public static int find_lis(int[] a) {
-	int n = a.length, lis = 0;
```
Comment (nwatx) [inline content/4_Gold/LIS.mdx:74]: why remove n?

**Review by nwatx — CHANGES_REQUESTED** [review]

**Review by dongliu0426 — APPROVED** [review]

**Review by TruaShamu — APPROVED** [review]

File: content/4_Gold/LIS.mdx (hunk 2 above)
Comment (SansPapyrus683) [reply content/4_Gold/LIS.mdx:74]: it's just redundant

**Pull request merged**